apps/molgenis-components/src/components/layout/CookieWall.vue

```diff
@@ -0,0 +1,115 @@
+<script setup>
+import { computed, ref } from "vue";
+import { useCookies } from "@vueuse/integrations/useCookies";
+import LayoutModal from "./LayoutModal.vue";
+import ButtonAction from "../forms/ButtonAction.vue";
+import ButtonAlt from "../forms/ButtonAlt.vue";
+import GTag from "./GTag.vue";
+const cookies = useCookies(["mg_allow_analytics"]);
+const props = defineProps({
+  title: {
+    type: String,
+    default: "Would you like a cookie ?",
```

Comment:
small thing, but maybe remove the space between cookie and ?

Reply:
done